packages/flutter/lib/src/material/arc.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:math' as math;
 import 'dart:ui' show hashValues, lerpDouble;
 
+import 'package:collection/collection.dart' show maxBy;
```

Comment:
Can we inline maxBy? We want to remove our dependency on this package if possible.